juriscraper/lib/string_utils.py

```diff
@@ -562,6 +562,16 @@ def normalize_dashes(raw_string):
     return raw_string
 
 
+def numbers_only(raw_string):
+    no_commas = re.sub(",", "", raw_string)
+    return list(
+        map(
+            lambda x: float(x) if "." in x else int(x),
+            re.findall("\d*\.?\d+|-\d*\.?\d+", no_commas),
+        )
+    )
```

Comment:
I'm not sure I understand why we need this. Maybe a docstring with an example? Looking at tests, this feels like overkill. Do we have document numbers with decimal points in them? That'd be news to me, I think.

Reply:
I built it to be general purpose. The biggest use case I was considering for decimals was dollar amounts that you want to pull out of a sentence for any reason. Couple options here:

1. I can make this for integers only.
2. I can add the doc string you mentioned.

Reply:
I wound up pulling this in the commit I just pushed. I didn't quite feel like we needed a general purpose solution for this yet. Thank you though.